.env

```diff
@@ -43,4 +43,5 @@ MESSENGER_TRANSPORT_DSN=doctrine://default?auto_setup=0
 ###< symfony/mailer ###
 ARTICLES_ON_PAGE=10
 PROGRAMME_MIN_TIME_MINUTES=15
-PROGRAMME_MAX_TIME_MINUTES=360
\ No newline at end of file
+PROGRAMME_MAX_TIME_MINUTES=360
+HTTP_IMPORT_ADDRESS=''
```

Comment:
Too vague. A proper name would be EVOZON_WAREHOUSE_BASE_URI.

You can have multiple import addresses and you need to be more specific.